src/integrator_interface.jl

```diff
@@ -1,137 +1,217 @@
-function savevalues!(integrator::DDEIntegrator,force_save=false)
-  integrator.integrator.u = integrator.u
-  integrator.integrator.k = integrator.k
-  integrator.integrator.t = integrator.t
-  OrdinaryDiffEq.ode_addsteps!(integrator.integrator,integrator.f)
-  savevalues!(integrator.integrator,force_save)
+"""
+    savevalues!(integrator::DDEIntegrator, force_save=false)
+
+Update solution of `integrator`, if necessary or forced by `force_save`.
```

Comment:
According to the Julia documentation guide lines, "In general, only the most generic method should be documented" - so maybe this (and other docstrings in `integrator_interface.jl`) should be moved to OrdinaryDiffEq.jl rather?